core/test/utilities/helpers/iframe-helpers.spec.js

```diff
@@ -327,16 +306,39 @@ describe('Iframe-helpers', () => {
       assert.equal(iframes.filter(i => i.active).length, 5, 'active iframes');
 
       const expectedKeys = ['id', 'container', 'active', 'type'];
-      assert.deepEqual(
-        Object.keys(iframes[0]),
-        expectedKeys,
-        'contains all required keys'
-      );
+      assert.deepEqual(Object.keys(iframes[0]), expectedKeys, 'contains all required keys');
 
       const mainIframes = IframeHelpers.getMainIframes('main');
       assert.equal(mainIframes.length, 2);
       const modalIframes = IframeHelpers.getModalIframes('modal');
       assert.equal(modalIframes.length, 1);
     });
   });
+  describe('applyCoreStateData', () => {
+    it('applyCoreStateData', () => {
+      sinon.stub(LuigiTheming, 'getCurrentTheme').returns('any');
+      sinon.stub(LuigiFeatureToggles, 'getActiveFeatureToggleList').returns(['featureToggle']);
+      sinon
+        .stub(LuigiI18N, 'getCurrentLocale')
+        .returns({ currentLocaleStorageKey: 'luigi.currentluigi', defaultLocale: 'luigi' });
+      const internalData = { context: 'luigi' };
```

Comment:
```suggestion
      const internalData = { context: 'luigi' };
      const expected = { 
          activeFeatureToggleList: ['featureToggle'],
          currentLocale: {
              currentLocaleStorageKey: 'luigi.currentluigi',
              defaultLocale: 'luigi'
          },
          currentTheme: 'any'
        }
```

Reply:
**Suggestion**: Let's define an `expected` object there. Avoid the repetition below.